lib/auth.js

```diff
@@ -220,6 +220,7 @@ function addViewVars(req, vars, callback) {
   vars.idp = {};
   vars.idp.identityBasePath = bedrock.config.idp.identityBasePath;
   vars.idp.identityBaseUri = vars.baseUri + vars.idp.identityBasePath;
+  vars.idp.owner = bedrock.config.idp.owner;
```

Comment:
Don't do this ... `bedrock.config.idp.owner` currently has the private key in it -- this exposes it to the frontend. Just expose what's needed (`id`?).